src/main/java/org/lightcouch/CouchDbClient.java

```diff
@@ -205,6 +206,8 @@ private PoolingHttpClientConnectionManager createConnectionManager(
 
 		if("https".equals(props.getProtocol())) {
 			if (props.isSSLAuthenticationDisabled()) {
+				// No SSL authentication.
+				// No need for a custom SSLSocketFactory in this case.
```

Comment:
Thanks for the notes w.r.t. bean-like expectations, I wasn't aware of those.

Thanks for adding the docs.